Doc/library/sqlite3.rst

```diff
@@ -47,84 +47,148 @@ This document includes four main sections:
       PEP written by Marc-André Lemburg.
 
 
+.. We use the following practises for SQL code:
+   - UPPERCASE for keywords
+   - SnakeCase for schema
+   - single quotes for string literals
+   - singular for table names
+   - if needed, use double quotes for table and column names
+
 .. _sqlite3-tutorial:
 
 Tutorial
 --------
 
-To use the module, start by creating a :class:`Connection` object that
-represents the database.  Here the data will be stored in the
-:file:`example.db` file::
+In this tutorial, you will learn the basics of the :mod:`!sqlite3` API
+by creating an on-disk movie database :file:`tutorial.db`
+an filling it with meta-data about Monty Python movies.
+A fundamental understanding of database concepts,
+like `transactions`_ and `cursors`_, is assumed.
+
+To start working with an SQLite database,
+you must first open a connection to it.
+Call the :func:`sqlite3.connect` function
+to create a connection to the database :file:`tutorial.db`::
 
    import sqlite3
-   con = sqlite3.connect('example.db')
+   con = sqlite3.connect("tutorial.db")
 
-The special path name ``:memory:`` can be provided to create a temporary
-database in RAM.
+SQLite will implicitly create an empty database file :file:`tutorial.db`
+in the current working directory if it does not exist.
+The returned :class:`Connection` object ``con``
+represents the connection to the on-disk database.
 
-Once a :class:`Connection` has been established, create a :class:`Cursor` object
-and call its :meth:`~Cursor.execute` method to perform SQL commands::
+In order to execute SQL statements and fetch results from SQL queries,
+use a database cursor.
+Call :meth:`~Connection.cursor` on ``con`` to create
+:class:`Cursor` ``cur``::
 
    cur = con.cursor()
 
-   # Create table
-   cur.execute('''CREATE TABLE stocks
-                  (date text, trans text, symbol text, qty real, price real)''')
+Now, create a database table ``movie`` with columns for title,
+year released, and `IMDB`_ review score.
+The `flexible typing`_ feature of SQLite makes typenames optional;
+for simplicity, just use column names in the table declaration.
+Execute the SQL statement
+by calling :meth:`~Cursor.execute` on ``cur``::
 
-   # Insert a row of data
-   cur.execute("INSERT INTO stocks VALUES ('2006-01-05','BUY','RHAT',100,35.14)")
+   cur.execute("CREATE TABLE Movie(Title, Year, Score)")
 
-   # Save (commit) the changes
-   con.commit()
+Verify that the table has been created by running a query against it.
```

Comment:
I was thinking about the tone of the whole tutorial.  To me imperative statements like this ("Verify...") sound a bit "cold" and demanding.  In general I prefer to use first person plural ("We can verify...") which gives the impression that we are following the reader step by step.  Second person singular ("You can verify...") is better then the bare imperative, but gives a bit the impression that the reader is on their own (which is kind of true, and not necessarily a bad thing).

Does Diataxis say anything about which person should we (you?) use?

Reply:
The tone is definitely something to consider. Imperative works well for the reference, but as you say, it may feel a little unwelcoming for a tutorial. If we decide to change the tone, let's do that after we've got all the other nits and details sorted out.